docs/built_rst/csv/elements/precinct.rst

```diff
@@ -127,33 +127,47 @@ external_geospatial_feature
++--------------------+-------------------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| feature_identifier | :ref:`multi-csv-feature-identifier` | **Required** | Repeats      | Identifiers indicating which specific    | If the element is invalid, then the      |
+|                    |                                     |              |              | shape(s) to use from the geospatial      | implementation is required to ignore the |
+|                    |                                     |              |              | file. These refer to identifiers within  | ``ExternalGeospatialFeature`` element    |
+|                    |                                     |              |              | the referenced external file. This is a  | containing it.                           |
+|                    |                                     |              |              | repeated field in the XML specification, |                                          |
+|                    |                                     |              |              | but a scalar field in the CSV            |                                          |
+|                    |                                     |              |              | specification. If more than one          |                                          |
+|                    |                                     |              |              | identifier is required with the CSV      |                                          |
+|                    |                                     |              |              | specifiation, multiple values can be     |                                          |
+|                    |                                     |              |              | provided by delimited by space.          |                                          |
++--------------------+-------------------------------------+--------------+--------------+------------------------------------------+------------------------------------------+
 
 .. code-block:: csv-table
    :linenos:
 
     id,external_file_id,file_format,shape_identifiers
     egf1,ef1,shp,0 7 9
+
+
+.. _multi-csv-feature-identifier:
+
+feature_identifier
+^^^^^^^^^^^^^^^^^^
+
++--------------+---------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| Tag          | Data Type     | Required?    | Repeats?     | Description                              | Error Handling                           |
++==============+===============+==============+==============+==========================================+==========================================+
+| index        | ``xs:string`` | Optional     | Single       | The index value for the shapefile        | If the field is invalid or not present,  |
```

Comment:
Looks like the CSV docs still need to be updated, the index field is still showing as a string type.